### PR TITLE
Ask on quora

### PR DESCRIPTION
**Pull request by samrudhisharma**
"Please review this, @casey-collab"

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1776)

### CODE REVIEW

File: english/question_on_quora.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Why not ask a question on Quora ?
```
Comment (casey-collab) [inline english/question_on_quora.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/ask-on-quora/ask-on-quora.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Why not ask a question on Quora ?
```
Comment (casey-collab) [inline english/ask-on-quora/ask-on-quora.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/question_on_quora.md (hunk 1 above)
Comment (casey-collab) [inline english/question_on_quora.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by samrudhisharma** [comment]
"Please review this, @casey-collab"

File: english/question_on_quora.md (hunk 1 above)
Comment (casey-collab) [inline english/question_on_quora.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/question_on_quora.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by samrudhisharma** [comment]
"Please review this, @casey-collab"

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**